src/sgame/sg_admin.cpp

```diff
@@ -6153,30 +6169,26 @@ static bool BotFillCmd( gentity_t *ent, const Cmd::Args& args )
 		return false;
 	}
 
-	for (team_t team : teams)
+	for (team_t t : teams)
```

Comment:
This change appeases the AppVeyor process. While this code isn't buggy this is a re-definition of `team` as I have already defined it at a higher scope.